src/main/java/com/mraof/minestuck/data/recipe/MinestuckRecipeProvider.java

```diff
@@ -97,6 +97,7 @@ protected void registerRecipes(Consumer<IFinishedRecipe> recipeBuilder)
 		ShapedRecipeBuilder.shapedRecipe(MSItems.CLOTHES_IRON).key('s', Tags.Items.STONE).key('i', Tags.Items.INGOTS_IRON).key('r', Tags.Items.RODS_WOODEN).patternLine(" rr").patternLine("sss").patternLine("iii").addCriterion("has_stick", hasItem(Tags.Items.RODS_WOODEN)).addCriterion("has_iron_ingot", hasItem(Tags.Items.INGOTS_IRON)).build(recipeBuilder);
 		ShapedRecipeBuilder.shapedRecipe(MSItems.HORN).key('w', Items.WHITE_WOOL).key('i', Tags.Items.INGOTS_IRON).key('r', Tags.Items.RODS_WOODEN).patternLine("  i").patternLine(" r ").patternLine("w  ").addCriterion("has_stick", hasItem(Tags.Items.RODS_WOODEN)).addCriterion("has_iron_ingot", hasItem(Tags.Items.INGOTS_IRON)).addCriterion("has_white_wool", hasItem(Items.WHITE_WOOL)).build(recipeBuilder);
 		ShapelessRecipeBuilder.shapelessRecipe(MSItems.BI_DYE, 2).addIngredient(Items.BLUE_DYE).addCriterion("has_blue_dye", hasItem(Items.BLUE_DYE)).addIngredient(Items.PINK_DYE).addCriterion("has_pink_dye", hasItem(Items.PINK_DYE)).addIngredient(Items.PURPLE_DYE).addCriterion("has_purple_dye", hasItem(Items.PURPLE_DYE)).build(recipeBuilder);
+		ShapedRecipeBuilder.shapedRecipe(MSItems.LIP_BALM).key('n', Items.IRON_NUGGET).key('w', Items.HONEYCOMB).patternLine("   ").patternLine("nwn").patternLine(" n ").addCriterion("has_honeycomb", hasItem(Items.HONEYCOMB)).addCriterion("has_iron_nugget", hasItem(Items.IRON_NUGGET)).build(recipeBuilder);
```

Comment:
I'd suggest removing the top pattern line.

Reply:
Done!